test/t8_test_init_linear_id.cxx

```diff
@@ -86,14 +85,14 @@ t8_num_descendants (t8_element_t * element, int level,
 /*Check, if all descendants of an element at level maxlvl have the same id on
  * the level of the input element as the input element*/
 static void
-t8_id_at_other_lvl_check (t8_element_t * element, t8_element * child,
-                          t8_eclass_scheme_c * ts, int maxlvl)
+t8_id_at_other_lvl_check (t8_element_t *element, t8_element * child,
+                          t8_eclass_scheme_c *ts, int maxlvl)
 {
   int                 level = ts->t8_element_level (element);
-  uint64_t            current_id = ts->t8_element_get_linear_id (element, level),
-                      num_descendants = t8_num_descendants (element, maxlvl, ts),
-                      id_at_lvl = ts->t8_element_get_linear_id (element, maxlvl),
-                      i, id;
+  uint64_t            current_id =
+    ts->t8_element_get_linear_id (element, level), num_descendants =
+    t8_num_descendants (element, maxlvl, ts), id_at_lvl =
+    ts->t8_element_get_linear_id (element, maxlvl), i, id;
```

Comment:
Please make this more readable by having a single declaration line per variable.
See  https://github.com/holke/t8code/wiki/Coding-Guideline#general-rules

Reply:
Done